ROADMAP.md

```diff
@@ -0,0 +1,43 @@
+# Roadmap
+
+# Phase 1 (v0.1.0):
+
+- [ ] Create a basic Bevy project and import the necessary dependencies.
+- [ ] Create a simple 3D scene with a camera and light.
+- [ ] Import a 3D model from a third-party software.
+- [ ] Display the 3D model in the scene.
+
+# Phase 2 (v0.2.0):
+
+- [ ] Implement a system for adding joints to imported models.
+- [ ] Implement a system for controlling joints with keyboard keys.
+- [ ] Add support for different types of joints (revolute, fixed, spherical, etc.).
+- [ ] Add support for attaching keyboard keys to motors.
+
+# Phase 3 (v0.3.0):
+
+- [ ] Implement more advanced features for controlling joints, such as speed and precision.
+- [ ] Add support for multiple joints in a single model.
+- [ ] Add support for nested joints.
+- [ ] Control spawn camera position and projection.
+
+# Phase 4 (v0.4.0):
+
+- [ ] Add support for more complex interactions between joints.
+- [ ] Add support for physics simulation of joints.
+- [ ] Add support for networking, so that multiple users can control the joints in a shared scene.
+
+# Phase 5 (v0.5.0):
+
+- [ ] Add additional features and polish, such as a user interface, documentation, and tests.
+
+# Phase 6 (v1.0.0):
+
+- [ ] Release the first stable version of the software.
+
+<!-- 
+Error handling: How will you handle errors that occur while loading or controlling the digital twins?
```

Comment:
Could you please explain the purpose of these four questions? :sweat_smile: Are those points of concern?

Reply:
Yes, they really are some concerns that we might think more about it in the future. I just left it there as reminder 😅